manifests/m/msys2/msys2/2022-01-18/msys2.msys2.installer.yaml

```
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: msys2.msys2
+PackageVersion: 2022-01-18
```

Version is without `-`
```suggestion
PackageVersion: 20220118
```